### DISCUSSION TITLE
[How] can I see past changes performed by WinGet?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [microsoft/winget-cli#4897] [Q&A] [answered]
As an example, for https://github.com/nextcloud/desktop/issues/7394#issue-2606416264, I would like to be able to ascertain when the package was last updated, and what version it updated from and to.

**Comment by Trenly** [comment, marked as answer]
Currently there is not an official tracking mechanism that is easy to access. Some installations make use of a file called `installed.db` which is located in WinGet AppData folders, but that is not guaranteed to have information about all applications or actions taken by WinGet and is for the CLI to better reason about applications that have been installed.

**Reply by roke-julian-lockhart** [reply]
> https://github.com/microsoft/winget-cli/discussions/4897#discussioncomment-11060371

@Trenly, do you think that an action log would be an acceptable FR?

**Reply by Trenly** [reply]
* https://github.com/microsoft/winget-cli/issues/3125
